gr-soapy/grc/soapy_sdrplay_source.block.yml

```diff
@@ -74,9 +88,15 @@ outputs:
 
 templates:
   imports: from gnuradio import soapy
-  make: |
-      None
-      dev = 'driver=sdrplay'
+
+  var_make: hello
+
+  make: |-
+      ## Negative value for agc_setpoint is placed in parens by GRC.
+      ## This code turns it into an int and bounds checks
+      _agc_setpoint = int(${agc_setpoint})
+      _agc_setpoint = max(min(_agc_setpoint, -20), -70)
+      dev = f'driver=sdrplay,agc_setpoint={_agc_setpoint}'
```

Comment:
I tried dev_args first and it didn't work. Maybe I was doing something wrong.

Reply:
The problem is that the user can specify dev_args. It would only take a coulple lines of code do merge them. This works, so I'll stick with this for now.